examples/github/schemas/github.schema.graphql

```diff
@@ -45029,3 +45029,7 @@ type WorkflowRun implements Node {
 A valid x509 certificate string
 """
 scalar X509Certificate
+
+scalar DuplicatedType
```

Comment:
since this one is actually github schema, I think better to update the basic example